README.md

```diff
@@ -350,34 +348,14 @@ Review all notable [changes](CHANGELOG.md#changelog) with the latest release
   + Ubuntu - `22.04` / `24.04`
   + RHEL - `8` / `9`
   + SLES - `15-SP5`
-* ROCm: `6.3.0`
+* ROCm: `7.0.0`
 * RPP - `2.0.0`
-* miopen-hip - `3.2.0.60300`
-* migraphx - `2.11.0.60300`
-* OpenCV - `4.6`
-* FFMPEG - [n4.4.2](https://github.com/FFmpeg/FFmpeg/releases/tag/n4.4.2)
+* miopen-hip - `3.4.0`
+* migraphx - `2.13.0`
+* OpenCV - `4.5.4`/`4.6`
+* FFMPEG - `4.4.2`
 * Dependencies for all the above packages
-* MIVisionX Setup Script - `V3.8.1`
+* MIVisionX Setup Script - `V4.0.0`
 
 ### Known issues
 * MIVisionX Package install in `RHEL`/`SLES` requires manual `OpenCV` and `FFMPEG` development packages installed
-
-## MIVisionX dependency map
```

Comment:
@kiritigowda  if you are removing this table, please remove the corresponding docker files also from ubuntu20/ and ubuntu22/ directories